packages/styles/scss/internal/content-item/_content-item.scss

```diff
@@ -101,6 +101,10 @@
     margin-block-end: $spacing-05;
   }
 
+  ::slotted(#{$c4d-prefix}-link-with-icon) {
```

Comment:
As [discussed in Slack](https://ibm-marketing-comms.slack.com/archives/C05P50B9UNT/p1734459607142819?thread_ts=1734352965.815569&cid=C05P50B9UNT), this selector is likely to miss the AEM scenario that came up, as originally reported in [ADCMS-6878](https://jsw.ibm.com/browse/ADCMS-6878), on [this test page](https://author-p131558-e1281330.adobeaemcloud.com/content/adobe-cms/us/en/carbon-v2-implementation-qa-devfolder/carbon-v2-implementation-qa-stagefolder/melike-test-stage/card-in-card--12-col.html?wcmmode=disabled).

Reply:
@m4olivei I moved that rule to a selector that was already selecting the slotted footer. Can you please check now?

Thanks!